src/sbm.jl

```diff
@@ -78,6 +78,12 @@
     temperature::Vector{T} | "°C"
     # Potential evapotranspiration [mm Δt⁻¹]
     potential_evaporation::Vector{T}
+    # Multiplication factor [-] for epot
+    epot_factor::Vector{T} | "-"
+    # Multiplication factor [-] for precip
+    precip_factor::Vector{T} | "-"
+    # Additive factor [-] for temp
+    temp_add::Vector{T} | "-"
```

Comment:
Regarding the naming of all three; I'd prefer to be consistent with the existing code as much as possible, so could you use the `<existing name>_<operation>` naming, where operation is scale or offset, and existing name is `potential_evaporation`, etc.

The same goes for the TOML and variable names below.

Reply:
agree, changed in 1a9c2c43cfc531832f5c56f4be854853992b03f4